python/tests/test_seldon_client.py

```diff
@@ -294,6 +327,25 @@ def test_predict_grpc_seldon():
     assert response.response.strData == "predict"
 
 
+@mock.patch("seldon_core.seldon_client.prediction_pb2_grpc.SeldonStub", new=MyStub)
+def test_predict_grpc_proto_raw_data_seldon():
+    sc = SeldonClient(deployment_name="mymodel", transport="grpc", gateway="seldon")
+    proto_raw_data = json_to_seldon_message(RAW_DATA_TEST)
+    response = sc.predict(raw_data=proto_raw_data, client_return_type="proto")
+    request = seldon_message_to_json(response.request)
+    assert request == RAW_DATA_TEST
+    assert response.response.strData == "predict"
```

Comment:
Interesting. Do you know where is this `strData` value coming from?

Reply:
Yepp - from the MyStub, line #279

Reply:
remove return value check